docs/faq.rst

```diff
@@ -78,6 +80,52 @@ If the journal you are publishing in requires you to use ``textsc`` for software
 
         \newcommand{\pyhf}{\textsc{pyhf}}
 
+Why use Python?
+~~~~~~~~~~~~~~~
+
+As of the late 2010's Python is widely considered the lingua franca of machine learning
+libraries, and is sufficiently high-level and expressive for physicists of various computational
+skill backgrounds to use.
+Using Python as the language for development allows for the distribution of the software
+--- as both source files and binary distributions --- through the Python Package Index (PyPI)
+and Conda-forge, which significantly lowers the barrier for use as compared to ``C++``.
+Additionally, a 2017 `DIANA/HEP <https://diana-hep.org/>`_ study :cite:`faq-feickert-diana-fellowship-report`
+demonstrated the graph structure and automatic differentiation abilities of machine learning
+frameworks allowed them to be quite effective tools for statistical fits.
+As the frameworks considered in this study (TensorFlow, PyTorch, MXNet) all provided
+low-level Python APIs to the libraries this made Python an obvious choice for a common
+high-level control language.
+Given all these considerations, Python was chosen as the development language.
+
+How did ``pyhf`` get started?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In 2017 Lukas Heinrich was discussing with colleauge Andy Buckley how it would be convienent to
```

Comment:
Andy Buckley -> Holger Schulz.  Maybe add that this resulted in paper w/ Jessica and Ye-Ling

Reply:
Revised. :+1: If you think the paper mention should get moved from where I put it for now,  can you make comments in the PR to indicate where?

Reply:
(I don't know the full history of this paper like you do, so this paper might have already been in progress when pyhf first got started and needs to get bumped up so it doesn't sound like a result _of_ pyhf).